sklearn/tests/test_isotonic.py

```diff
@@ -508,3 +508,37 @@ def test_make_unique_dtype():
         w = np.ones_like(x)
         x, y, w = _make_unique(x, y, w)
         assert_array_equal(x, [2, 3, 5])
+
+
+def test_isotonic_2darray_1_feature():
```

Comment:
Seems like the test is more general than that so I would suggest `test_input_shape_validation`

Reply:
done